scriptworker/task.py

```diff
@@ -556,12 +556,15 @@ def prepare_to_run_task(context, claim_task):
         int: exit code
 
     """
+    env = deepcopy(os.environ)
+    env['TASK_ID'] = context.task_id or 'None'
```

Comment:
As a more general question, is the environment the only way that we can communicate between `scriptworker` and the task script?

Reply:
`env['TASK_ID']` is the standard way of communicating the taskId from the worker to the task across worker implementations. Scriptworkers *also* have the ability to read `task.json` and other metadata from disk, since scriptworker writes those artifacts in a predictable place the script can read. I think implementing the same convention that generic- and docker-worker have with the env var makes sense here.